app/services/teacher_training_public_api/sync_courses.rb

```diff
@@ -49,6 +51,10 @@ def create_or_update_course(course_from_api, recruitment_cycle_year, incremental
         SetOpenOnApplyForNewCourse.new(course).call
```

Comment:
could we move the open course logic into the `SetOpenOnApplyForNewCourse` service?

Reply:
The SetOpenOnApplyForNewCourse checks the course for being in the previous cycle and being open on apply as part of the call and handles slack messaging - I didnt think those parts were neccesary. Additionally the name would have to change. Seeing as the only really duplicated bit of logic is the open! method I didnt see it as particularly related (unless im missing something).